packages/app-store/vital/api/callback.ts

```diff
@@ -34,7 +35,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
         },
       },
     });
-    return res.redirect("/apps/installed");
+    return res.redirect(getInstalledAppPath({ variant: "other", slug: "vital-automation" }));
```

Comment:
This is to support redirecting to the correct category within the Installed Apps page once the app has been installed.